enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -35,53 +35,70 @@ The purpose of log forwarding is to provide a declarative way by which adopters
-* It is not a goal for the tech-preview to support log forwarding outputs other then the ones identified for the goals. Admins can forward to their own fluentd via `forward` 
-and then configure that fluentd to forward to any number of specific logstore outputs
-* It is not a goal to provide secure storage for audit logs. If the deployer chooses to enable audit log forwarding, they need to make sure that the endpoint is compliant with governmental regulations and secure. The OpenShift logging Elasticsearch does not comply with those regulations.
+* No secure storage for audit logs, only secure (TLS) delivery to a target system.
+  - The user must ensure that the target system is secure and compliant with regulations.
+  - The OpenShift Elasticsearch store is not guaranteed to comply with any such regulations.
+* No direct access to the configuration schemes of target systems or the local collector
+  - limited access to essential output-specific features, e.g. setting a syslog facility.
+* Not intended to provide a complex routing solution as one might achieve by using a custom collector configuration or a messaging solution (e.g. kafka) - but is intended to allow forwarding to an such a system that is deployed externally.
 
 ## Proposal
 
-Log forwarding will provide a declarative way to specify the outputs for specific types of logs using a 'pipeline'.  A 'pipeline' defines simple routing for one source to one or more outputs. The source of logs are opinionated and well defined by cluster logging. The initial source types are as follows:
+Provide a declarative `pipeline` that associates a set of named `inputs` with a set of named `outputs`.
+
+The following reserved input names are defined for the initial release:
+* `Application` - Container logs generated by user applications running on the platform, excluding `Infrastructure` containers.
+* `Infrastrucure` - Logs generated by infrastructure components running on the platform and OKD nodes (e.g. journal logs).  "Infrastructure" applications are defined as any pods which run in namespaces: `openshift*`, `kube*`, `default`.
+* `Audit` - Logs generated by the nodes' auditd (/var/log/audit/audit.log), audit logs from the kubeapi-server and the openshift-apiserver. This will not be forwarded by default.
 
-* `logs.app` - Container logs generated by user applications running on the platform, excluding infrastructure container applications
-* `logs.infra` - Logs generated by both infrastructure components running on the platform and OKD nodes (e.g. journal logs).  "Infra" applications are defined as any pods which run in namespaces: `openshift*`, `kube*`, `default`.
-* `logs.audit` - Logs generated by the nodes' auditd (/var/log/audit/audit.log), audit logs from the kubeapi-server and the openshift-apiserver. This will not be forwarded by default.
+The following reserved output names are defined:
+* `Default` - the current Elasticsearch based store.
```

Comment:
add detail of `Default` output ?

```
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogForwarder"
spec:
  outputs:
   - name: Default
     type: Elasticsearch
     url: tls://${elasticsearch-url}:9200
     secret:
        name: es_secrets # Must contain keys tls.key, tls.cert and ca.cert
```

Reply:
should users even be able to specify anything if they're using `Default` ?

Reply:
just for documentation purposes, wanted to show the definition of a predefined output `Default` in this proposal.

Reply:
Agreed that the documentation should show such examples. The "Default" name in configuration lets the user request our default *without* knowing the details, and its actual meaning might change from release to release.